### PR TITLE
github actions: staging website

### PR DESCRIPTION
**Pull request by jmfayard**
https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -64,7 +64,7 @@ nav:
   - 'Setup': setup.md
   - 'Add Dependencies': add-dependencies.md
   - 'Update Dependencies': update-dependencies.md
-#  - 'Gradle buildSrcVersions': gradle-buildsrcversions.md
+  - 'Gradle buildSrcVersions': gradle-buildsrcversions.md
```
Comment (LouisCAD) [inline mkdocs.yml:67]: This change is unrelated to this PR (out of the scope), plus we previously agreed that we would not make it part of the main menu for now.

Comment (LouisCAD) [inline mkdocs.yml:67]: This change is unrelated to this PR (out of the scope), plus we previously agreed that we would not make it part of the main menu for now.

File: site.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# USAGE:
+#    site.sh              # serve website locally
+#    site.sh mybranch     # publish to https://jmfayard.github.io/rvdev/
+#    site.sh release      # publish to https://jmfayard.github.io/refreshVersions/
```
Comment (LouisCAD) [inline site.sh:5]: The word "site" lacks explicitness about what it actually does.

I think this should be split into two scripts:
1. `build-website.sh`
2. `publish-website.sh`

The `publish-website.sh` script would:
1. Require a branch name to be passed as an argument.
2. Run the `build-website.sh` script.
3. Publish it at the right place.

**Review by LouisCAD — CHANGES_REQUESTED** [review]

**Comment by jmfayard** [comment]
the GitHub actions do not even run for unknown reasons so I gave up on this pull request

**Comment by jmfayard** [comment]
marked as draft

**Pull request closed without merging**